src/image-map.ts

```diff
@@ -89,10 +89,10 @@ export const icons = {
     "v1706616417/subject-icons/design-technology.svg",
   "subject-drama": "v1706616417/subject-icons/drama.svg",
   "subject-english": "v1706616417/subject-icons/english.svg",
+  "subject-english-literature": "v1706616421/subject-icons/literature.svg",
+  "subject-english-language": "v1706616419/subject-icons/language.svg",
   "subject-english-grammar": "v1706616417/subject-icons/english-grammar.svg",
   "subject-english-handwriting": "v1706616419/subject-icons/handwriting.svg",
-  "subject-english-reading-for-pleasure":
```

Comment:
I'd probably leave this in, incase it's used by anybody else. Otherwise it'd be a breaking change